8Knot/app.py

```diff
@@ -89,67 +77,18 @@
     background_callback_manager=celery_manager,
 )
 
-# expose the application object's server variable so that the wsgi server can use it.
+"""CONFIGURE FLASK-LOGIN STUFF"""
```

Comment:
A little more informative comment

Reply:
What should be commented here vs. in the configuration function itself? The comment is more of a section-delineator than informative.

Reply:
Looks good and makes sense. the "STUFF" threw me off a bit/ wanted to make sure it wasnt an development comment that needed more context